src/SimpleDiffEq.jl

```diff
@@ -12,6 +12,7 @@ using RecursiveArrayTools
 
 include("functionmap.jl")
```

Comment:
export it?

Reply:
I exported the symbol `SimpleRK4`, should I export `simplerk4_init` also?